changes/1414-silversurfer34.md

```diff
@@ -0,0 +1,2 @@
+Fix proposal for the issue 1414, wrong behavior of json method when there are nested models with custom root.
+Once the dict is generated, it is parsed and each time an item has the __root__ key, its associated value is attached to the parent key
```

Comment:
please use back quotes around code or code like values in markdown, here you'll get "has the __root__ key" not "has the `__root__` key"

Reply:
done, I did not knew the back quote trick thank you